src/main/kotlin/org/rust/ide/formatter/blocks/RsMacroArgFmtBlock.kt

```diff
@@ -23,11 +29,51 @@ class RsMacroArgFmtBlock(
     override fun getIndent(): Indent? = indent
     override fun getWrap(): Wrap? = wrap
 
-    override fun getSubBlocks(): List<Block> = emptyList()
+    override fun getSubBlocks(): List<Block> = mySubBlocks
+    private val mySubBlocks: List<Block> by lazy { buildChildren() }
 
-    override fun getSpacing(child1: Block?, child2: Block): Spacing? = Spacing.getReadOnlySpacing()
-    override fun getChildAttributes(newChildIndex: Int): ChildAttributes = ChildAttributes(null, null)
+    private fun buildChildren(): List<Block> {
+        return node.getChildren(null)
+            .filter { !it.isWhitespaceOrEmpty() }
+            .map { childNode: ASTNode ->
+                RsFormattingModelBuilder.createBlock(
+                    node = childNode,
+                    alignment = null,
+                    indent = computeIndentForChild(childNode),
+                    wrap = null,
+                    ctx = ctx)
+            }
+    }
+
+    override fun getSpacing(child1: Block?, child2: Block): Spacing? {
+        return Spacing.getReadOnlySpacing()
```

Comment:
Btw this code actually prevents formatter from doing its work, hence formatter does not work in macros. But if we'll decide to enable it, we can just replace this line with `return null`. Formatter may be handy in macros e.g. when copy-pasting a code into macros.